index.html

```diff
@@ -1,61 +1,174 @@
+                <div class = "product-list__choosen-products--wrapper">
+                <li
+                  class="product-list__item product-list__item--choosen-products"
+                >
+                  <div class="product-list__title--choosen-products">
+                    Apple:
+                  </div>
+
+                  <div class="product-list__description--choosen-products">
+                    150g ·
+                  </div>
+                  <div class="product-list__description--choosen-products">
+                    78kcal
+                  </div>
+                </li>
+
+                <li
+                  class="product-list__item product-list__item--choosen-products"
+                >
+                  <div class="product-list__title--choosen-products">Egg:</div>
+                  <div class="product-list__description--choosen-products">
+                    100g ·
+                  </div>
+                  <div class="product-list__description--choosen-products">
+                    140kcal
+                  </div>
+                </li>
+
+                <li
+                  class="product-list__item product-list__item--choosen-products"
+                >
+                  <div class="product-list__title--choosen-products">
+                    Orange:
+                  </div>
+                  <div class="product-list__description--choosen-products">
+                    250g ·
+                  </div>
+                  <div class="product-list__description--choosen-products">
+                    197.5kcal
+                  </div>
+                </li>
+
+                <li
+                  class="product-list__item product-list__item--choosen-products"
+                >
+                  <div class="product-list__title--choosen-products">
+                    Chicken and rice:
+                  </div>
+                  <div class="product-list__description--choosen-products">
+                    300g ·
+                  </div>
+                  <div class="product-list__description--choosen-products">
+                    237kcal
+                  </div>
+                </li>
+
+                <li
+                  class="product-list__item product-list__item--choosen-products"
+                >
+                  <div class="product-list__title--choosen-products">
+                    Banana yoghurt:
+                  </div>
+                  <div class="product-list__description--choosen-products">
+                    150g ·
+                  </div>
+                  <div class="product-list__description--choosen-products">
+                    129kcal
+                  </div>
+                </li>
+              </div>
+              </ul>
+            </div>
+
+            <div class="section-box__wrapper--right">
+              <h3 class="section-box__title--right">Total:</h3>
+              <div class="total__label--wrapper">
```

Comment:
Abstrahując od tego, że po zmianie z komentarza niżej drugi człon w  `total__label` będzie inny, ten div nie powinien mieć klasy `total__label--wrapper`, bo 
- nie ma klasy bazowej, na którą byś aplikował modyfikator (`total__label total__label--wrapper`)
- nazwa `--wrapper` sugeruje, że jest to element, nie modyfikator

Także musisz wymyślić takie nazwy klas, żeby zachować semantykę ;)